scripts/asm-installer/asm_vm

```diff
@@ -1110,6 +1110,23 @@ this project, run:
   --project="${PROJECT_ID}"
 EOF
     fi
+
+    info "Verifying GCE image..."
+    local SOURCE_IMAGE_PATH="$(echo "$(retry 2 run gcloud compute instance-templates \
```

Comment:
You don't need `echo` here.

Reply:
we need echo here to extract the contents and work with the jq

Reply:
`retry` will output the intended result, so you can directly use jq with it. e.g. https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages/blob/master/scripts/asm-installer/install_asm#L1968